P/Prjtrellis/build_tarballs.jl

```diff
@@ -0,0 +1,46 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Prjtrellis"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/YosysHQ/prjtrellis.git", "dff1cbcb1bd30de7e96f8a059f2e19be1bb2e44d")
+]
+
+dependencies = [
+    Dependency("Python_jll"),
```

Comment:
What is python used for?

Reply:
There are python bindings via `pytrellis`. Some of the downstream packages (PRs incoming) use this for interface.

Reply:
```suggestion
    Dependency("Python_jll"; compat="~3.8.1"),
```